<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page1_start</class>
 <widget class="QMainWindow" name="page1_start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>page1_start</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/images/page1_back.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>430</width>
      <height>450</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="next_pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>340</y>
       <width>200</width>
       <height>82</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="welcome_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>345</width>
       <height>141</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>100</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="logo_label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>300</width>
       <height>132</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>300</width>
       <height>132</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/images/1.test.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
